internal/querynode/flow_graph_service_time_node.go

```diff
@@ -64,15 +62,9 @@ func (stNode *serviceTimeNode) Operate(in []flowgraph.Msg) []flowgraph.Msg {
 	}
 
 	// update service time
-	var id UniqueID
-	if stNode.loadType == loadTypePartition {
-		id = stNode.partitionID
-	} else {
-		id = stNode.collectionID
-	}
-	err := stNode.tSafeReplica.setTSafe(stNode.vChannel, id, serviceTimeMsg.timeRange.timestampMax)
+	err := stNode.tSafeReplica.setTSafe(stNode.vChannel, serviceTimeMsg.timeRange.timestampMax)
 	if err != nil {
-		log.Warn(err.Error())
+		log.Error(err.Error())
```

Comment:
It is recommended to have a contextual description when printing the log

Reply:
description has been wrapped in tSafeReplica

Reply:
fixed